app/src/main/AndroidManifest.xml

```diff
@@ -31,6 +31,7 @@
     <application
             android:name=".WireApplication"
             android:allowBackup="false"
+            android:fullBackupContent="false"
```

Comment:
Yup, we were looking with @MohamadJaara  and seems to prevent some issues regarding lost Keys and Encrypted shared prefs. So it will not hurt to add it =)